server/sonar-project.properties

```diff
@@ -0,0 +1,15 @@
+sonar.organization=dedis
+sonar.projectKey=hbt-server
```

Comment:
question: do we want a single project for the SMC and the blockchain, or two separate projects ?

Reply:
I can't take a decision to change this right now. Maybe I need a coffee break :-)

Reply:
as agreed, keep one project for now.